guides/authentication.md

```diff
@@ -125,6 +125,7 @@ async function quickstart() {
   const client = new CogniteClient({
     appId: 'Cognite SDK samples',
     project,
+    baseUrl: "https://api.cognitedata.com",
```

Comment:
there is a baseUrl variable further up, perhaps reusing that would avoid any unexpected behavior if anyone runs this =)

Reply:
You mean the one defined in the **OIDC authentication using code authorization w pkce** snippet?

Reply:
whops, seems i scrolled a bit too quickly through this